docs/source/contributing/developing-core.md

```diff
@@ -9,7 +9,17 @@ myst:
 
 # Develop Volto core
 
-This chapter describes how to develop Volto core and its libraries, packages, and apps as open source software contributions.
+This chapter describes how to develop the latest version of Volto core and its libraries, packages, and apps as open source software contributions.
+
+```{seealso}
+For previous versions of Volto core, you should follow the guide in the relevant version branch to build and run the backend.
+
+-   [Volto 17](https://github.com/plone/volto/blob/17.x.x/api/README.rst)
```

Comment:
Isn't specifying versions here a bad thing? It may become outdated and we may forget to update it.

Reply:
Not really. When we do any major release, we have to find all references to version numbers in all docs and update them anyway.

Where possible, we would update version numbers to use either includes or replacements. See the main docs configuration and https://github.com/plone/documentation/blob/6.0/docs/conf.py#L350-L356 usage https://github.com/search?q=repo%3Aplone%2Fdocumentation%20SUPPORTED_PYTHON_VERSIONS&type=code.

That will make maintenance easier going forward.

I also have plans to make 6.docs.plone.org the stable version and main.docs.plone.org the new unstable development version. But I need to talk more with the release team about how to implement this, as it has a lot of moving parts.

Reply:
@stevepiercy I wouldn't mention 15, it's been long gone and no longer supported. I would mention only 16.x.x and 17.x.x

Reply:
@stevepiercy the link https://github.com/plone/volto/blob/17.x.x/api/README.rst is only for backend installation, not for "Develop Volto core".

Reply:
> @stevepiercy the link https://github.com/plone/volto/blob/17.x.x/api/README.rst is only for backend installation, not for "Develop Volto core".

Oh, you explain this in "you should follow the guide in the relevant version branch to build and run the backend". So ok!

Reply:
Removed 15 in 772ca5a52d835d72620fc61c6ac354af533df694.